gui/reporting/rrd.py

```diff
@@ -250,6 +251,54 @@ def graph(self):
         return args
 
 
+class CPUTempPlugin(RRDBase):
+
+    title = "CPU Temperature"
+    vertical_label = "�C"
+
+    cpu_count = 0
+
+    def __init__(self, base_path, identifier=None, unit=None, step=None):
+        super(CPUTempPlugin, self).__init__(base_path, identifier, unit, step)
+                
+        proc = pipeopen('/sbin/sysctl -n kern.smp.cpus', important=False, logger=log)
+        proc_out = proc.communicate()[0]
+        try:
+            self.cpu_count = int(proc_out)
+        except:
+            self.is_available = False
```

Comment:
I dont understand how having a kern.smp.cpus sysctl is the key for it having the CPU Temperature report working or not.